src/index.ts

```diff
@@ -67,10 +69,12 @@ const listrTasks = [
 
   // Linting
   eslint,
+  stylelint,
   prettier,
```

Comment:
should be this sort alphabetically?

i think having stylelint check after eslint in the preflight output make sense but I would need another oppinion

Reply:
it's fine (short list), let's leave it as you have it